src/GetSystemMetrics-Hook.cpp

```diff
@@ -38,28 +38,29 @@ VOID WINAPI MyExitProcess(UINT uExitCode)
 }
 
 // Install all detour hooks
-void InstallDetourHooks()
+static void InstallDetourHooks()
 {
     DetourTransactionBegin();
     DetourUpdateThread(GetCurrentThread());
 
-    // Hook GetSystemMetrics
-    if (DetourAttach(reinterpret_cast<PVOID *>(&Original_GetSystemMetrics), Hooked_GetSystemMetrics) != NO_ERROR)
+    struct HookPair
     {
-        DetourTransactionAbort();
-        return;
-    }
-    // Hook TerminateProcess
-    if (DetourAttach(reinterpret_cast<PVOID *>(&Original_TerminateProcess), MyTerminateProcess) != NO_ERROR)
-    {
-        DetourTransactionAbort();
-        return;
-    }
-    // Hook ExitProcess
-    if (DetourAttach(reinterpret_cast<PVOID *>(&Original_ExitProcess), MyExitProcess) != NO_ERROR)
+        PVOID *ppOriginal;
+        PVOID pHook;
+    };
+
+    HookPair hooks[] = {
+        {reinterpret_cast<PVOID *>(&Original_GetSystemMetrics), reinterpret_cast<PVOID>(Hooked_GetSystemMetrics)},
+        {reinterpret_cast<PVOID *>(&Original_TerminateProcess), reinterpret_cast<PVOID>(Hooked_TerminateProcess)},
+        {reinterpret_cast<PVOID *>(&Original_ExitProcess), reinterpret_cast<PVOID>(MyExitProcess)}};
```

Comment:
This is used here and in `UninstallDetourHooks`, so could we have this be module global (aka `static const` at the top of the file)?